omod/src/main/webapp/resources/scripts/conditionlist/controllers/manageconditions.controller.js

```diff
@@ -40,8 +40,16 @@ function ManageConditionsController($scope, RestfulService, CommonFunctions) {
         }
 
     self.removeCondition = self.removeCondition || function () {
-            $scope.conditionToBeRemoved.voided = true;
-            $scope.conditionToBeRemoved.endDate = new Date();
+            
+var conditions = [];
+var condition;
+
+      for(condition in conditions){
+       condition.voided = true;
+        condition.endDate = new Date();
+        console.log(condition);
```

Comment:
this will actualy log nothing, your Condiitons List is , empty , it points to nothing